cmds/core/sync/sync.go

```diff
@@ -28,39 +30,42 @@ var (
 	filesystem = flag.Bool("filesystem", false, "commit filesystem caches to disk")
 )
 
-func init() {
-	flag.BoolVar(data, "d", false, "")
-	flag.BoolVar(filesystem, "f", false, "")
+var usage = "Usage: %s [OPTION] [FILE]...\n"
 
-	flag.Usage = func() {
-		fmt.Fprintf(os.Stderr, "Usage: %s [OPTION] [FILE]...\n", os.Args[0])
-		flag.PrintDefaults()
-	}
+func init() {
+	util.Usage(usage)
 }
 
-func doSyscall(syscallNum uintptr) {
-	for _, fileName := range flag.Args() {
+func doSyscall(syscallNum uintptr, args []string) error {
+	for _, fileName := range args {
 		f, err := os.OpenFile(fileName, syscall.O_RDONLY|syscall.O_NOCTTY|syscall.O_CLOEXEC, 0o644)
 		if err != nil {
-			log.Fatal(err)
+			return err
 		}
 		_, _, err = syscall.Syscall(syscallNum, uintptr(f.Fd()), 0, 0)
-		if err != nil {
-			log.Fatal(err)
+		if errors.Is(err, fs.ErrNotExist) {
```

Comment:
why change from `err != nil` to `errors.Is(err, fs.ErrNotExist)`? please change back?

Reply:
syscall.Syscall gives back an errno err which can't be nil. So i wanted to check if there is no error.
But it has to be: if !errors.Is(err, fs.ErrNotExist)
Please correct me if i am wrong.